MaterialGraphProject/Assets/UnityShaderEditor/Editor/Data/Nodes/Artistic/Blend/BlendNode.cs

```diff
@@ -44,270 +44,312 @@ protected override MethodInfo GetFunctionToConvert()
         static string Unity_Blend_Burn(
             [Slot(0, Binding.None)] DynamicDimensionVector Base,
             [Slot(1, Binding.None)] DynamicDimensionVector Blend,
-            [Slot(2, Binding.None)] out DynamicDimensionVector Out)
+            [Slot(2, Binding.None)] Vector1 Opacity,
```

Comment:
Annoying but can you please give these default values of 1 (every Opacity slot)
`[Slot(2, Binding.None, 1, 1, 1, 1)] Vector1 Opacity`